### PR TITLE
First pass at styles

### PR DESCRIPTION
**Pull request by odrusso**
Before:
Screen Shot 2021-09-10 at 9 17 18 PM

After:
Screen Shot 2021-09-11 at 2 14 52 PM

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "url": "git+https://github.com/odrusso/eduace.git"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.10",
```
Comment (odrusso) [inline web/package.json:16]: Not sure why so many packages got updated :/
Just did added some dev packages lol

File: web/src/components/App.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap');
+
+$base-font: 'Open Sans', sans-serif;
+
+$bg-color: rgb(245, 246, 250);
+$eduace-gray: rgb(160, 160, 160);
```
Comment (ashtonmoomoo) [inline web/src/components/App.scss:6]: ™️

File: web/src/components/App.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.eduace-app-container {
+  display: grid;
+  grid-template-columns: repeat(10, 10%);
+  min-height: 100vh;
+}
+
+.eduace-content-container {
+  grid-column: 3 / 10;
```
Comment (ashtonmoomoo) [inline web/src/components/App.scss:24]: I haven't seen this grid stuff before but if it works how I think it does, that's super cool

Comment (odrusso) [reply web/src/components/App.scss:24]: HTML5 Grid is the future (until the next big thing) aye

**Review by ashtonmoomoo — APPROVED** [review]

**Pull request merged**